docs/root/intro/arch_overview/security/google_vrp.rst

```diff
@@ -1,45 +1,48 @@
-.. _arch_overview_bug_bounty:
+.. _arch_overview_google_vrp:
 
-Bug bounty
-==========
+Google Vulnerability Reward Program (VRP)
+=========================================
 
-In the near future, Envoy will have a bug bounty program. This will be open to all security
-researchers and will provide rewards for vulnerabilities discovered and reported according
-to the rules below.
+Envoy is a participant in `Google's Vulnerability Reward Program (VRP)
+<https://www.google.com/about/appsecurity/reward-program/>`_. This is open to all security
+researchers and will provide rewards for vulnerabilities discovered and reported according to the
+rules below.
 
-.. note::
-  This is not yet funded, stay tuned and reach out to envoy-security@googlegroups.com for further
-  details.
-
-.. _arch_overview_bug_bounty_rules:
+.. _arch_overview_google_vrp_rules:
 
 Rules
 -----
 
-The goal of this bug bounty is to provide a formal process to honor contributions from external
+The goal of the VRP is to provide a formal process to honor contributions from external
 security researchers to Envoy's security. Vulnerabilities should meet the following conditions
 to be eligible for the program:
 
 1. Vulnerabilities must meet one of the below :ref:`objectives
-   <arch_overview_bug_bounty_objectives>`, demonstrated with the supplied Docker-based
-   :ref:`execution environment <arch_overview_bug_bounty_ee>` and be consistent with the
-   program's :ref:`threat model <arch_overview_bug_bounty_threat_model>`.
+   <arch_overview_google_vrp_objectives>`, demonstrated with the supplied Docker-based
+   :ref:`execution environment <arch_overview_google_vrp_ee>` and be consistent with the
+   program's :ref:`threat model <arch_overview_google_vrp_threat_model>`.
 
 2. Vulnerabilities must be reported to envoy-security@googlegroups.com and be kept under embargo
    while triage and potential security releases occur. Please follow the :repo:`disclosure guidance
    <SECURITY.md#disclosures>` when submitting reports. Disclosure SLOs are documented :repo:`here
-   <SECURITY.md#fix-and-disclosure-slos>`.
+   <SECURITY.md#fix-and-disclosure-slos>`. In general, security disclosures are subject to the
+   `Linux Foundation's privacy policy <https://www.linuxfoundation.org/privacy/>`_ with the added
+   proviso that VRP reports (including reporter e-mail address and name) may be freely shared with
+   Google for VRP purposes.
 
 3. Vulnerabilities must not be previously known in a public forum, e.g. GitHub issues trackers,
    CVE databases (when previously associated with Envoy), etc. Existing CVEs that have not been
    previously associated with an Envoy vulnerability are fair game.
 
-Rewards are at the discretion of the Envoy OSS security team and will be conditioned on the above
-criteria. If multiple instances of the same vulnerability are reported at the same time by
+4. Vulnerabilities must not be also submitted to a parallel reward program run by Google or
```

Comment:
@sirdarckcat here's the point I promised on double submissions.

Reply:
looks good! thank you